### PR TITLE
add sept email per bug 1400968

### PR DESCRIPTION
**Pull request by peiying2**
Polish only

### CODE REVIEW

File: en-US/emails/2017/fx_mobile_update.lang [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## NOTE: Email is about the Pocket feature built in to Firefox.
+## NOTE: Pocket as a brand remains in English.
+## NOTE: Bug 1400968; Polish only
+
+
+# Subject Line + Preheader
+;What’s new with Firefox mobile
+What’s new with Firefox mobile
+
```
Comment (flodolo) [inline en-US/emails/2017/fx_mobile_update.lang]: Fix the weird spacing between strings

File: en-US/emails/2017/fx_mobile_update.lang [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# Headline
+;Our goal is simple: make Firefox Mobile as awesome as possible, and make sure you know all the awesome things you can do with it. Look out for the Firefox Mobile Update in your in-box whenever we have exciting stuff to share.
+Our goal is simple: make Firefox Mobile as awesome as possible, and make sure you know all the awesome things you can do with it. Look out for the Firefox Mobile Update in your in-box whenever we have exciting stuff to share.
+
+# Introduction
+;Sure, you COULD have a million tabs open or an inbox full of articles you emailed yourself. Or you could just click Pocket.
```
Comment (flodolo) [inline en-US/emails/2017/fx_mobile_update.lang]: Where is this string coming from? The headline from the spreadsheet is missing.

Comment (peiying2) [reply en-US/emails/2017/fx_mobile_update.lang]: argh - jump the tab to a different email!

File: en-US/emails/2017/fx_mobile_update.lang [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+;Multi-task on Firefox Focus for Android
+Multi-task on Firefox Focus for Android
+
+
+# Module 2
+;You asked for multiple tabs. We’ve answered with Multi-Tasking, a clean, focused way of browsing several sites at once.
+You asked for multiple tabs. We’ve answered with Multi-Tasking, a clean, focused way of browsing several sites at once.
+
+
+# Moduel 2
```
Comment (flodolo) [inline en-US/emails/2017/fx_mobile_update.lang]: typo: module

**Comment by flodolo** [comment]
> Add the pl file directly on master

I added it directly, so it should show up in Pontoon in a couple of minutes.

**Pull request merged**